llvm/lib/Target/RISCV/GISel/RISCVInstructionSelector.cpp

```diff
@@ -509,19 +495,19 @@ static void getOperandsForBranch(Register CondReg, MachineRegisterInfo &MRI,
 bool RISCVInstructionSelector::select(MachineInstr &MI) {
   MachineBasicBlock &MBB = *MI.getParent();
   MachineFunction &MF = *MBB.getParent();
-  MachineRegisterInfo &MRI = MF.getRegInfo();
+  MRI = &MF.getRegInfo();
   MachineIRBuilder MIB(MI);
```

Comment:
Unrelated, but should not create single use MachineIRBuilders. Generally shouldn't need it during selection anyway

Reply:
Since it is unrelated, I will address this in a different PR. It looks like AArch64 creates a significant number of these MachineIRBuilders. And it looks like both RISC-V and AArch64 need to use it during instruction selection. Could you please tell me more about what you have in mind?